readme.md

```diff
@@ -2,13 +2,73 @@
 
 > One stop shop to document your react components.
 
+<p align="center">
+  <img width="500" src="./assets/example.png">
```

Comment:
This link should work, not sure why it isn't coming up in preview mode. 
Might need to merge the PR first?

Reply:
maybe
TBH i usually just upload it in a github comment and then paste the URL